Sources/StreamChatUI/ChatChannelList/ChatChannelUnreadCountView.swift

```diff
@@ -47,7 +52,7 @@ open class _ChatChannelUnreadCountView<ExtraData: ExtraDataTypes>: _View, UIConf
     }
     
     override open func updateContent() {
-        isHidden = content.mentionedMessages == 0 && content.messages == 0
+        isHidden = shouldBeHidden
```

Comment:
How do we need two properties? Can't we just have:
```
open var isHidden: Bool {
        content.mentionedMessages == 0 && content.messages == 0
    }
```

Reply:
Hum I see, isHidden is from UIView. But the naming here is a bit confusing and redundant.
I think this is enough: 
```
isHidden = content == .noUnread
```
We don't need an extra property. Because if the customer can access `isHidden` and `shouldBeHidden`, what that means? Simply accessing the `.content` property and check that it is `noUnread` is enough